vip-cache-manager/vip-cache-manager.php

```diff
@@ -26,45 +36,43 @@ function __construct() {
 		add_action( 'init', array( $this, 'init' ) );
 	}
 
-	function init() {
+	public function init() {
 		if ( is_super_admin() && isset( $_GET['cm_purge_all'] ) && check_admin_referer( 'manual_purge' ) ) {
```

Comment:
I'm going to address this in a separate PR, to avoid scope expansion here and get this changeset deployed 😃